canvas/cli/assignment/fetch.py

```diff
@@ -32,10 +33,15 @@ def main():
 def _modify_parser(parser):
     parser.description = 'Fetch information for an assignment.'
 
-    parser.add_argument('-t', '--table', dest = 'table',
+    group = parser.add_mutually_exclusive_group()
+
+    group.add_argument('-t', '--table', dest = 'table',
         action = 'store_true', default = DEFAULT_TABLE,
         help = 'Output the results as a TSV table with a header (default: %(default)s).')
 
+    group.add_argument('--json', action = 'store_true',
+        help = 'Output in JSON format instead of TSV')
+
```

Comment:
Instead of adding these args in all parsers,
you can make some common code that adds these to a passed in parser.
Something like `canvas.cli.common.add_output_args(parser)`.